spec/workspace-spec.js

```diff
@@ -2528,6 +2528,62 @@ i = /test/; #FIXME\
     })
   })
 
+  describe('::activateNextPane', () => {
+    describe('when the active workspace pane is inside a dock', () => {
```

Comment:
This looks great. Tiny nitpick that doesn't matter that much... I think you could just say "pane" instead of "workspace pane"